### PR TITLE
Feature/immutable teaching call comments

### PR DESCRIPTION
**Pull request by jaroldwong**
Card: https://trello.com/c/WKv7sEoS/2154-2-update-instructor-comments-on-the-teaching-call-form-to-be-a-series-of-immutable-comments-ala-budget-comments

### CODE REVIEW

File: app/assignment/directives/modals/commentModal/commentModal.css [hunk 1]
```diff
@@ -8,5 +8,17 @@
 }
 
 .comment-modal__comment {
-  white-space: pre;
+  display: flex;
+	justify-content: space-between;
```
Comment (ltwheeler) [inline app/assignment/directives/modals/commentModal/commentModal.css]: Something funny happened here with indentation? File should be 2 spaces?

File: app/assignment/directives/modals/commentModal/commentModal.html [hunk 2]
```diff
@@ -4,7 +4,10 @@
       No Instructor comments.
     </div>
 
-    <div class="comment-modal__comment" ng-show="instructorComment.length > 0">{{ instructorComment }}</div>
+    <div class="comment-modal__comment" ng-repeat="comment in instructorComment | orderBy: '-lastModifiedOn'">
```
Comment (ltwheeler) [inline app/assignment/directives/modals/commentModal/commentModal.html]: Whats the - do?
Is that a reverse order flag? :D

File: app/teachingCall/teachingCallForm/templates/TeachingCallForm.html [hunk 3]
```diff
@@ -183,10 +183,31 @@ <h5 class="inline-block">Unavailabilities</h5>
...
 			<div class="col-md-12">
 				<h5>Comments</h5>
-				<textarea class="form-control teaching-call-form-comments" 
-				ng-model="view.state.comment" auto-input on-blur="updateComment()"
-				placeholder="(Optional) Add any additional comments or requests here."
-				rows="5" id="comment"></textarea>
+				<div class="teaching-call-form-comment-input-container">
+					<div class=" teaching-call-form-comment-input-textarea">
+						<textarea class="form-control" 
+											ng-model="view.state.newComment" auto-input
```
Comment (ltwheeler) [inline app/teachingCall/teachingCallForm/templates/TeachingCallForm.html]: Something funny happened with indentation here and below

File: app/assignment/directives/modals/commentModal/commentModal.html (hunk 2 above)
Comment (jaroldwong) [reply app/assignment/directives/modals/commentModal/commentModal.html]: Yup, sorts item in descending order. Seems to work fine in this simple use case.

File: app/assignment/directives/modals/commentModal/commentModal.css (hunk 1 above)
Comment (jaroldwong) [reply app/assignment/directives/modals/commentModal/commentModal.css]: I might have copy a line from somewhere and mixed up tabs and spaces. It's fixed to 2 spaces now.

**Pull request merged**